XamlControlsGallery/ControlPages/ExpanderPage.xaml

```diff
@@ -0,0 +1,83 @@
+                    <ComboBox x:Name="ExpandDirectionComboBox" Header="ExpandDirection" SelectionChanged="ExpandDirectionComboBox_SelectionChanged" SelectedValue ="Down">
+                        <x:String>Down</x:String>
+                        <x:String>Up</x:String>
+                    </ComboBox>
+                </StackPanel>
+            </local:ControlExample.Options>
+
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;muxc:Expander
+     IsExpanded="$(IsExpanded)"
+     ExpandDirection ="$(ExpandDirection)" 
+     Header="This text is in the header"
+     Content="This is in the content"/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="IsExpanded" Value="{x:Bind Expander1.IsExpanded, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="ExpandDirection" Value="{ x:Bind ExpandDirectionComboBox.SelectedValue, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+        
+        <local:ControlExample x:Name="Example2" HeaderText="An Expander with other controls in the header and content">
+            <local:ControlExample.Example>
+                <muxc:Expander x:Name="Expander2" IsExpanded="False" ExpandDirection ="Down">
+                    <muxc:Expander.Header>
+                        <ToggleButton>This is a ToggleButton in the header</ToggleButton>
+                    </muxc:Expander.Header>
+                    <muxc:Expander.Content>
+                        <Button>This is a Button in the content</Button>
+                    </muxc:Expander.Content>
+                </muxc:Expander>
+            </local:ControlExample.Example>
+            <!--<local:ControlExample.Options>
+                <StackPanel>
+                    <ComboBox x:Name="ExpandDirectionComboBox" Header="ExpandDirection" SelectionChanged="ExpandDirectionComboBox_SelectionChanged" SelectedValue ="Down">
+                        <x:String>Down</x:String>
+                        <x:String>Up</x:String>
+                    </ComboBox>
+                </StackPanel>
+            </local:ControlExample.Options>
+
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;muxc:Expander
+     IsExpanded="$(IsExpanded)"
+     ExpandDirection ="$(ExpandDirection)" 
+     Header="This text is in the header"
+     Content="This is in the content"/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="IsExpanded" Value="{x:Bind Expander1.IsExpanded, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="ExpandDirection" Value="{ x:Bind ExpandDirectionComboBox.SelectedValue, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>-->
```

Comment:
Please delete. Better yet, please update and uncomment the ControlExample.Xaml section.